code/nodes/runModels/funcs.q

```diff
@@ -0,0 +1,143 @@
+\d .automl
+
+// Definitions of the main callable functions used in the application of runModels
+
+// @kind function
+// @category runModels
+// @fileoverview Set value of random seed for reproducability
+// @param cfg {dict} configuration information relating to the current run of AutoML
+// @return {Null} Value of seed is set
+runModels.setSeed:{[cfg]
+  system"S ",string cfg`seed;
+  }
+
+// @kind function
+// @category runModels
+// @fileoverview Apply train test split to keep holdout for feature impact plot and testing of vanilla best model
+// @param cfg  {dict} Configuration information relating to the current run of AutoML
+// @param tts  {dict} Feature and target data split into training and testing set
+// @return {dict} Training and holdout split of data
+runModels.holdoutSplit:{[cfg;tts]
+  ttsFunc:get cfg`tts;
+  ttsFunc[tts`xtrain;tts`ytrain;cfg`hld]
+  }
+
+// @kind function
+// @category runModels
+// @fileoverview Seeded cross-validation function, designed to ensure that models will be consistent
+//  from run to run in order to accurately assess the benefit of updates to parameters
+// @param tts  {dict} Feature and target data split into training and testing set
+// @param cfg  {dict} Configuration information relating to the current run of AutoML
+// @param mdl  {tab}  Model to be applied to feature data
+// @return {(bool[];float[])} Predictions and associated actual values for each cross validation fold
+runModels.xValSeed:{[tts;cfg;mdl]
+  xTrain:tts`xtrain;
+  yTrain:tts`ytrain;
+  scoreFunc:cfg[`prf]mdl`minit;
+  seedModel:`seed~mdl`seed;
+  isSklearn:`sklearn~mdl`lib;
+  // Seed handled differently for sklearn and keras  
+  seed:$[not seedModel;
+      ::;
+    isSklearn;
+      enlist[`random_state]!enlist cfg`seed;
+      (cfg`seed;mdl`typ)
+      ];
+  $[seedModel&isSklearn;
+    // Grid search required to incorporate the random state definition
+    [gsFunc:get cfg[`gs]0;
+     numFolds:cfg[`gs]1;
+     numReps:cfg[`gs]2;
```

Comment:
Not really sure on customising the number of repetitions, the system is already slow on large datasets, with gs/xv the major bottleneck.